<launch>
	<!-- Create an instance of each appropriate node -->
  	<node name="raptor_mcom" pkg="raptor" type="raptor_mcom" respawn="false" output="screen"/>
	<node name="raptor_mcom_testbench" pkg="raptor" type="raptor_mcom_testbench" respawn="false" output="screen"/>
</launch>

